Userland/Libraries/LibWeb/XHR/XMLHttpRequest.cpp

```diff
@@ -578,8 +598,11 @@ DOM::ExceptionOr<void> XMLHttpRequest::send(String body)
 
     auto request = LoadRequest::create_for_url_on_page(request_url, m_window->page());
     request.set_method(m_method);
-    if (!body.is_null())
-        request.set_body(body.to_byte_buffer());
+    if (!body_with_type.get<0>().is_empty()) {
+        request.set_body(body_with_type.get<0>());
+        if (!body_with_type.get<1>().is_empty())
+            request.set_header("Content-Type", body_with_type.get<1>());
+    }
```

Comment:
This is really awkward. Can we just use a two-element struct for this? :)

Reply:
Agree, I have changed it to a struct. The spec speaks about a Tuple, though for the sake of readability should probably not interpret the spec exact word by word.